daemon/info.go

```diff
@@ -174,7 +174,10 @@ func (daemon *Daemon) fillSecurityOptions(v *types.Info, sysInfo *sysinfo.SysInf
 	if sysInfo.Seccomp && supportsSeccomp {
 		profile := daemon.seccompProfilePath
 		if profile == "" {
-			profile = "default"
+			profile = config.SeccompProfileDefault
+		}
```

Comment:
Should this be normalized in the option parsing instead of being normalized in the info output?

Reply:
(That would also cut down on the number of `... != "" && ... != Default` dual comparisons.)

Reply:
Added a commit that moves this logic to `setupSeccompProfile()`